projects/go-style-guide/src/app/features/ui-kit/components/modal-docs/modal-docs.component.ts

```diff
@@ -12,6 +12,11 @@ export class ModalDocsComponent {
 
   pageTitle: string = 'Modal';
 
+  componentBindings: string = `
+  @Input() modalTitle:  string      = '';
+  @Input() modalSize:   'lg' | 'xl' = 'lg';
```

Comment:
For consistency sake on the docs should we not line up the '=' here?

Reply:
I noticed on some doc pages we line up the `=` and on some we don't. If we've agreed on not lining them up, then I will change it.